### PR TITLE
Lurker no longer decloaks if they pounce onto a dead guy.

### PR DESCRIPTION
**Pull request by TheGamerdk**
# About the pull request

Doesn't make sense to lose your cloak for pouncing at a dead guy.

They'll still decloak if it's a living guy hiding under a dead guy.

(This only happens if your pounce ends on a tile where there's a dead guy)

# Explain why it's good for the game
Inconsistent at best. Bad

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
fix: Lurkers no longer lose their pounce if they happen to end their pounce on a tile with a dead human
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_abilities.dm [hunk 1]
```diff
@@ -21,6 +21,8 @@
 	if (X.mutation_type == LURKER_NORMAL)
 		var/found = FALSE
 		for (var/mob/living/carbon/human/H in get_turf(X))
+			if(H.stat == DEAD)
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_abilities.dm:24]: Please also fix the single letter vars here. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#variables
You can use VSC to do this easily by selecting the function, search, enable dropdown for replace, enable word, enable case, and then for H/X and replace with `human`/`xeno` or something.

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by ihatethisengine** [comment]
Discord broke indentations

**Comment by TheGamerdk** [comment]
> Discord broke indentations

I saw

**Review by Drulikar — APPROVED** [review]
Changes look good. I even test merged with the new weeded bodies to make sure the interaction was the same.

**Pull request merged**